django_slack/api.py

```diff
@@ -8,7 +8,10 @@
 from .app_settings import app_settings
 
 def slack_message(template, context=None, attachments=None, fail_silently=None, **kwargs):
-    backend = get_backend()
+    # expect backend to be a str or unicode if present; keep the name for re-assign
+    backend = kwargs.pop('backend', None)
+    channel = kwargs.pop('channel', None)
```

Comment:
If we wrap the ``None`` with ``app.settings.CHANNEL`` we can remove the ``or`` later